bulletin-de-la-societe-entomologique-de-france.csl

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Bulletin de la Société Entomologique de France</title>
```

Comment:
@KwAntynn, I assume it's okay if I change the title to `<title>Bulletin de la Société entomologique de France</title>`? (the publisher doesn't seem to uppercase the "entomologique")

We'd also want to add the language to the title (`<title>Bulletin de la Société entomologique de France (French)</title>`).